community.hachyderm.io/content/en/blog/incidents/2023-07-16-security-redis-cache-exposed/index.md

```diff
@@ -0,0 +1,77 @@
+---
+date: 2023-05-16
+title: "community.hachyderm.io/content/en/blog/incidents/2023-02-27-tls-media"
+linkTitle: "community.hachyderm.io/content/en/blog/incidents/2023-02-27-tls-media"

```

Comment:
Should be changed to `Security Incident: Hachyderm’s Redis Cache Exposed to Public Internet`

Reply:
fixed in separate pr